core/cmd/forwarders_commands_test.go

```diff
@@ -76,7 +75,7 @@ func TestShell_TrackEVMForwarder(t *testing.T) {
 	set := flag.NewFlagSet("test", 0)
 	cltest.FlagSetApplyFromAction(client.TrackForwarder, set, "")
 
-	require.NoError(t, set.Set("address", "0x5431F5F973781809D18643b87B44921b11355d81"))
+	require.NoError(t, set.Set("address", utils.RandomAddress().Hex()))
```

Comment:
Won't this create a new address?

Reply:
It does, yes -- is that a concern?

Reply:
Isn't this supposed to be the same address as above? So it should be `address.Hex()`?

Reply:
Ahh, now I see that we are not testing for that, disregard